### PR TITLE
fix: Explain that the volume sync doesn't apply to boot volumes

### PR DESCRIPTION
**Pull request by fghaas**
In the how-to guide on synchronizing volumes with minimal downtime, explain that the process does not apply for boot volumes (and suggest to retype those volumes offline, instead).

### CODE REVIEW

**Review by colder-is-better — DISMISSED** [review]
LGTM!

**Comment by fghaas** [comment]
Force-pushed this with no changes in order to rekick the CI pipeline, hoping that this time around the CI status is being reported correctly. Once the checks pass, we'll need a re-approval to be able to merge this.

**Review by colder-is-better — APPROVED** [review]

**Pull request merged**
